### PR TITLE
Make the readme a little more markup and have a navigation around the table of contents.

### PR DESCRIPTION
**Pull request by UberMeatShield**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
 * Ensure the access_token checks id_token.exp, iss, etc
 * Restrict the access_token update to only in keep_alive
+* Demonstrate session refreshing in [resty-openidc](https://github.com/zmartzone/lua-resty-openidc)
+* Show how to do token refreshing with [MSAL.js](https://github.com/AzureAD/microsoft-authentication-library-for-js)
 
-## Web / Azure Keepalive frontend
+
+Services
+--------
 There are two docker containers one respresents a webserver that is around to show how to proxy the connection from auth, display login information, AND show the keep alive method using MSAL.js.
```
Comment (bac) [inline README.md]: You mention two containers but seem to only discuss one. What is the other one?

That first sentence is a run on.  

"There are two docker containers. One..."  Alternatively use a colon, but those are so fancy.

File: README.md [hunk 2]
```diff
@@ -16,19 +45,17 @@ into.
 * The webserver runs on http://localhost:8000 and has one endpoint that is open access called /heartbeat
 which is the kind of thing you need in order to work with a load balancer (aws).
 
-## Azure Authentication using OpenResty lua-resty-openidc and lua-resty-jwt
-
+Authentication
+--------------
```
Comment (bac) [inline README.md:48]: Funny, I prefer the `#` syntax of markdown.  Whatever works!

**Review by bac — APPROVED** [review]
Great README Justin!

File: README.md (hunk 1 above)
Comment (UberMeatShield) [reply README.md]: Modified to describe the containers better.

**Pull request merged**